ui/demo/less/reader.less

```diff
@@ -60,32 +60,17 @@ body {
 
 
 .reader__popover__bbox {
-  border: 1px transparent dotted;
+  border: 1px transparent dashed;
 
-  &.rotate0 {
-    border-bottom-color: @dark-grey
-  }
-  
-  &.rotate90 {
-    border-left-color: @dark-grey;
-  }
-
-  &.rotate180 {
-    border-top-color: @dark-grey;
+  &.rotate0, &.rotate90, &.rotate180, &.rotate270 {
```

Comment:
If the same border color is applied regardless of how the bounding box is rotated, do you need to specify the rotation at all? It looks like you could move the border color up into `reader__popover__bbox`.

Reply:
Isn't we want to display the border only if we rotate instead of by default @carolinepaulic

Reply:
okay cool considered it done